packages/browser/src/integrations/breadcrumbs.ts

```diff
@@ -325,7 +325,35 @@ function addSentryBreadcrumb(serializedData: string): void {
         event,
       },
     );
-  } catch (_oO) {
-    logger.error('Error while adding sentry type breadcrumb');
+  } catch (error) {
+    logger.error('Error while adding sentry type breadcrumb will try envelope', error);
```

Comment:
This will add a lot of noise when we know envelopes are not "exceptional behavior".

What if we explicitly selected whether to try to decode `application/x-sentry-envelope` vs `application/json`, and then log a potential error as before?

There must be a way to distinguish the two without parsing all input. Or maybe a way for `addSentryBreadcrumb` to get the event object directly instead of a serialized version of it.

As it is implemented now, this error path requires deserializing the event and maybe that's not necessary?